File: java/src/main/java/it/gabrieletondi/telldontaskkata/usecase/SellItemRequest.java
package it.gabrieletondi.telldontaskkata.usecase;

public class SellItemRequest {
    private final int quantity;
    private final String productName;

    public SellItemRequest(String productName, int quantity) {
        this.quantity = quantity;
        this.productName = productName;
    }

    public int getQuantity() {
        return quantity;
    }

    public String getProductName() {
        return productName;
    }
}

File: typescript/src/useCase/SellItemRequest.ts
class SellItemRequest {
  constructor(name: string, quantity: number) {
    this.productName = name;
    this.quantity = quantity;
  }

  private quantity: number;
  private productName: string;

  public getQuantity(): number {
    return this.quantity;
  }

  public getProductName(): string {
    return this.productName;
  }
}

export default SellItemRequest;
